### PR TITLE
Change ssh key to ECDSA  from RSA for eos appliance

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gosriniv@redhat.com>

This change is made to support FIPS mode.

### CODE REVIEW

File: zuul.d/arista-eos-jobs.yaml [hunk 1]
```diff
@@ -6,6 +6,9 @@
     parent: ansible-network-appliance-base
     pre-run: playbooks/ansible-network-eos-appliance/pre.yaml
     run: playbooks/ansible-network-eos-appliance/run.yaml
+    vars:
+      zuul_ssh_key_algorithm: ecdsa
+      zuul_ssh_key_size: 521
```
Comment (pabelanger) [inline zuul.d/arista-eos-jobs.yaml:11]: We won't do this any more. The fix will be to update base-minimal-test to generate both rsa / ecdsa keys so both keys are on disk

Comment (pabelanger) [reply zuul.d/arista-eos-jobs.yaml:11]: https://github.com/ansible/project-config/blob/master/playbooks/base-minimal-test/pre.yaml#L15

if you want to propose a PR to remove our fork, that is step 1

**Comment by GomathiselviS** [comment]
recheck

File: zuul.d/project.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@
             files:
               - playbooks/ansible-network-appliance-base/.*
               - playbooks/ansible-network-asa-appliance/.*
+            voting: false
```
Comment (pabelanger) [inline zuul.d/project.yaml:11]: if you revert this, the jobs won't run

File: zuul.d/project.yaml [hunk 3]
```diff
@@ -51,11 +52,6 @@
               - playbooks/ansible-test-network-integration-base/files/bootstrap-splunk.yaml
               - playbooks/ansible-network-appliance-base/.*
               - roles/splunk.*
-        - ansible-security-trendmicro-deepsec-appliance:
-            files:
```
Comment (pabelanger) [inline zuul.d/project.yaml:55]: same

File: zuul.d/project.yaml [hunk 4]
```diff
@@ -74,6 +70,7 @@
             files:
               - playbooks/ansible-network-appliance-base/.*
               - playbooks/ansible-network-asa-appliance/.*
+            voting: false
```
Comment (pabelanger) [inline zuul.d/project.yaml:73]: same

**Comment by pabelanger** [comment]
lets revert your changes, to zuul config. The jobs won't run, now because your base-minimal-test change is gone.

**Pull request merged**